lib/chef/knife/bootstrap.rb

```diff
@@ -28,6 +28,7 @@ class Knife
     class Bootstrap < Knife
       include DataBagSecretOptions
 
+      CHEF_15 ||= 15
```

Comment:
I don't know that this constant is really helpful, but since it is a constant you could do `15.freeze`

Reply:
I think, it can be revemod safely.